exosphere-shared/src/docker-helper.ls

```diff
@@ -9,6 +9,12 @@ class DockerHelper
     child_process.exec-sync('docker ps -a --format {{.Names}}', 'utf8') |> (.includes container)
 
 
+  @ensure-container-is-running = (container, image) ->
```

Comment:
This is the name of the container, and not the container itself, right? Let's call this variable `container-name`.